### DISCUSSION TITLE
How to enter displaymode inside a colorbox?

### DISCUSSION THREAD

**Discussion by SymboLinker** [KaTeX/KaTeX#3226] [Q&A] [answered]
When using `\colorbox{#ADD8E6}{\( .. \)}`, or `\colorbox{#ADD8E6}{$ .. $}`, mathmode is used inside the colorbox, but having displaymode inside a colorbox seems not supported, or am I missing something?

Because display mode can be used via `$$ .. $$` or `\[ .. \]`, I have tested both `\colorbox{#ADD8E6}{\[ .. \]}` and `\colorbox{#ADD8E6}{$$ .. $$}`.

As colorbox content for testing, you could use `\sum_{\square }^{\square }`, resulting (for mathmode) in:

![image](https://user-images.githubusercontent.com/32139898/131780792-f15e0a0a-0b8e-42bc-abdb-13348603ff68.png)

I would like to have be able to set the following in a colorbox as well:

![image](https://user-images.githubusercontent.com/32139898/131780982-048834f1-d4b7-4b8c-a552-975345da83ae.png)

**Comment by edemaine** [comment, marked as answer]
LaTeX and KaTeX offer `\displaystyle` to switch to display style math (and `\textstyle` and `\scriptstyle`).

So try this: `\colorbox{#ADD8E6}{$\displaystyle .. $}`

**Reply by SymboLinker** [reply]
Perfect. Thanks for your time!
